DataFormats/HcalRecHit/interface/HBHERecHit.h

```diff
@@ -18,9 +18,9 @@ class HBHERecHit : public CaloRecHit {
   HBHERecHit(const HcalDetId& id, float amplitude, float timeRising, float timeFalling=0);
 
   /// get the hit falling time
-  float timeFalling() const { return timeFalling_; }
+  inline float timeFalling() const { return timeFalling_; }
   /// get the id
-  HcalDetId id() const { return HcalDetId(detid()); }
+  inline HcalDetId id() const { return HcalDetId(detid()); }
```

Comment:
There is only one inline statement, which replaced a non-inline
statement. This normally makes the code faster, bit it is still
up to the compiler whether to really inline. Since the body
is very short, there are no disadvantages to making this function
inline.

________________________________
From: David Lange [notifications@github.com]
Sent: Thursday, July 14, 2016 1:48 AM
To: cms-sw/cmssw
Cc: Volobouev, I; Mention
Subject: Re: [cms-sw/cmssw] Initial implementation of the HB/HE Phase 1 reconstruction (#15092)


In DataFormats/HcalRecHit/interface/HBHERecHit.h<https://github.com/cms-sw/cmssw/pull/15092#discussion_r70756111>:

>    /// get the id
> -  HcalDetId id() const { return HcalDetId(detid()); }
> +  inline HcalDetId id() const { return HcalDetId(detid()); }


@igv4321<https://github.com/igv4321> do these two inline statements actually have an effect?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/cms-sw/cmssw/pull/15092/files/cb95544ae57dc7aa6cab5481015b80942b61d6d8#r70756111>, or mute the thread<https://github.com/notifications/unsubscribe/AFS4-WLGK48WoUxWabdz8kfb0yv5mr-9ks5qVdvPgaJpZM4JCc97>.